### ISSUE TITLE
Progression

### ISSUE THREAD

**Issue by ShaVerZ** [123Venom/plugin.video.venom#32]
I have a title which is in the unfinished section (Movies) at 0% but I cannot remove it.

If i mark it as watched in Trakt then it disappears but as soon as i add it back to my watchlist, it automatically goes back in to the unfinished section at 0%.

I've deleted all cache within settings, restart Kodi and the title remains as is.

Any ideas how I can resolve this please? 

v3.4.5

**Comment by bjbca** [comment]
I would log into Trakt through a web browser and try clean it up there, then start kodi and try update from there and see what happens.

**Comment by 123Venom** [comment]
I agree with @bjbca suggestion.  Try that and report back.  If it's still an issue I'll have to look at the progress in the meta response from Trakt api to see what's up.

**Comment by ShaVerZ** [comment]
I thought I had updated this post, apologies. I thought it would be a good idea to post the resolution I found, hopefully this will help others.

Trakt Support say that resetting and progression of movies/episodes is not supported via the website and it is only supported via the APi.

However, I found a solution to my problems. Visit the following website to manage your Trakt Progression;

[Trakt.tv Progression Manager](https://sharkykh.github.io/tppm/)
